easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-0.9.0-foss-2019b.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'CMakeMakeCp'
+
+name = 'RAxML-NG'
+version = '0.9.0'
+
+homepage = "https://github.com/amkozlov/raxml-ng"
+description = """RAxML-NG is a phylogenetic tree inference tool which
+uses maximum-likelihood (ML) optimality criterion. Its search heuristic
+ is based on iteratively performing a series of Subtree Pruning and Regrafting
+ (SPR) moves, which allows to quickly navigate to the best-known ML tree."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Comment:
the toolchain can probably be lowered to `gompi`, since this doesn't seem to depend on the linear algebra libraries